### DISCUSSION TITLE
sso支持

### DISCUSSION THREAD

**Discussion by yangzehan** [apache/streampark#3657] [Q&A] [answered]
根据官网给出的示例打开sso支持
![image](https://github.com/apache/incubator-streampark/assets/76773988/d437bcb4-3415-4770-80b3-860b45010ec1)
![image](https://github.com/apache/incubator-streampark/assets/76773988/0189807a-0d22-4c64-b187-27f599806574)
![image](https://github.com/apache/incubator-streampark/assets/76773988/278251ec-3f82-47ea-a900-0ef6fe1dad2d)

这是我的github上的oauth设置
![image](https://github.com/apache/incubator-streampark/assets/76773988/249ffa4d-04ac-4c14-82bc-7577504cd465)
本地调试
启动程序
![image](https://github.com/apache/incubator-streampark/assets/76773988/d780e91c-0888-4920-9fc9-e8112619ca6e)
可以看到已经有配置信息了
打开前端登录页面

![image](https://github.com/apache/incubator-streampark/assets/76773988/3d376abc-b445-44bb-9e2b-a9346d4ed707)
点击sso登录按钮
![image](https://github.com/apache/incubator-streampark/assets/76773988/c5bbcf99-f4fa-4bc7-b08b-b89fff7f5914)
页面没有挑转，请问是哪一步的配置有问题呢？

**Comment by saLeox** [comment]
Hi, 可以看看浏览器console或者后台日志，看是否有任何的输出或者报错？

**Reply by yangzehan** [reply]
![image](https://github.com/apache/incubator-streampark/assets/76773988/ff89a909-25c8-4807-bec4-900504a37284)
在我刚刚刷新的时候 跳转到了github的认证界面,然后我再次重新从主页点击sso登录按钮后又出现无法跳转的问题  看控制台感觉没有什么异常
![Uploading image.png…]()

**Reply by saLeox** [reply]
可以尝试在无痕模式中再次登陆，如果已经跳到GitHub页面，请尝试完成整个登陆流程，看是否可以顺利跳转回菜单页。

**Reply by yangzehan** [reply]
在我发出这个提问后我挂着网页没有刷新，然后当看到你的回复后我刷新了一下网页就跳转到了github的认证页面，但是我再次从登录页想点击sso登录发现又是无法跳转

**Reply by saLeox** [reply, marked as answer]
建议清空缓存或者开启无痕模式，完成完整登陆。

**Reply by yangzehan** [reply]
我尝试更换为Google浏览器  前后端重新启动后还是无法正常跳转,如果方便的话可以加个联系方式远程调试试试吗

**Reply by saLeox** [reply]
可以留下所在的开发群和群昵称

**Reply by yangzehan** [reply]
不好意思我还没有进群 我再尝试添加微信还没通过

**Reply by yangzehan** [reply]
a627617031是我的微信号,微信名是蟑螂恶霸

**Comment by caicancai** [comment]
Since sensitive information such as WeChat is involved, closing
